examples/grpc/security-abac/src/main/resources/application.conf

```diff
@@ -0,0 +1,91 @@
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+grpc {
```

Comment:
There is no hard rule for this yet, but we should be consistent with the configuration snippets showed in the docs and examples.

I would prefer to show-case YAML everywhere.

@tomas-langer @barchetta  thoughts ?

Reply:
This file should actually have been deleted when the config was changed to yaml for all gRPC examples. I can only think that either a rebase or merge somehow brought it back. I have removed it now.

Reply:
Still here